Simulation_engine/simulate_dotations.py

```diff
@@ -45,7 +45,7 @@ def format_reforme_openfisca(reforme_a_traduire):
         # on choisit de ne rien soulever si ce champ n'est pas présent dans la réforme.
         # Ca fait qu'une absence de ce paramètre ne fera pas échouer la requête.
         pass
-    return {"dgf": translate_dict(ref, TABLE_LEXIMPACT_TO_OFDL)}
+    return {"dgf": translate_dict(ref, TABLE_LEXIMPACT_TO_OFDL, leave_not_found=False)}
```

Comment:
This will IGNORE extra fields and NOT raise an error. I think it's a cool behaviour but I clarify.

Reply:
It's more that we are ignoring parts than inferring. leave_not_found is actually supposed to describe the behaviour :  should keys not found in the translation be "left" in the result or removed :)